.github/workflows/ci.yml

```diff
@@ -194,7 +194,9 @@ jobs:
       - name: Install Cypress
         run: yarn cypress install
 
-      - name: Cypress run
+      - env:
+          DEBUG: '@cypress/github-action'
+        name: Cypress run
```

Comment:
I've found Cypress tests to start to be flaky recently and seemingly more so on this branch. But, of course, as soon as I added the commit with these debug logs, everything passed. 🙃 

I'm hoping we'll get more info on what's going wrong as new commits enter this branch...